test/ApiTestCase.php

```diff
@@ -101,6 +101,29 @@ abstract class ApiTestCase extends TestCase
         );
     }
 
+    final protected function mockArticleVoRCall(string $id, ArticlePoA $article)
```

Comment:
seems to stub the VoR call but is passed a PoA and returns a similar `Content-Type`. Perhaps all is matching this use case but consider making it uniform?